### PR TITLE
Add magazines to 9mm weapons

### PR DESCRIPTION
**Pull request by mugling**
Continues directly from #14995 and updates all 9mm handguns to use magazines as per #14981. I'm stuck on what do to differentiate the remaining SMGs so suggestions are welcomed. I know @chaosvolt is working on a craftable sten magazine which seems appropriate given that weapons design.

Combined with #14995 this provides a good example of ammo conversions - the USP 9mm pistol with a .45 ACP conversion then loads using USP 45 mags. The Leadworks magazines behave similarly.

We should probably penalise ammo conversion more to offset their much improved utility in allowing you to use guns for which you currently lack the appropriate magazines.

### CODE REVIEW

**Comment by chaosvolt** [comment]
Ah, nice. And yeah, regarding pistol/SMG magazines the PR I mentioned has mags for the Makarov, both versions of the Skorpion, the Thompson, and the Sten.

File: data/json/items/magazines.json [hunk 1]
```diff
@@ -25,35 +81,183 @@
...
+        "volume" : 2,
+        "weight" : 210,
+        "ammo_type" : "9mm",
+        "capacity" : 30,
+        "reliability" : 9,
+        "reload_time" : 100,
+        "flags" : [ "COMPACT_MAG" ]
+    },
+    {   "type" : "MAGAZINE",
+        "id" : "mp5mag",
```
Comment (chaosvolt) [inline data/json/items/magazines.json]: Same ID as standard MP5 magazine.

Comment (mugling) [inline data/json/items/magazines.json]: Thanks, will fix.

File: data/json/items/magazines.json [hunk 2]
```diff
@@ -8,12 +8,68 @@
...
-        "to_hit" : -2,
+        "weight" : 360,
         "ammo_type" : "223",
         "capacity" : 30,
-        "reliability" : 10,
-        "reload_time" : 30
+        "reliability" : 8,
+        "reload_time" : 100,
+        "flags" : [ "COMPACT_MAG" ],
+        "alternatives" : [
```
Comment (chaosvolt) [inline data/json/items/magazines.json]: Now you have me confused as to whether this property goes on magazine entries, or firearm entries.

Comment (mugling) [inline data/json/items/magazines.json]: My bad in #15002, you do indeed at the `alternatives` to magazines

**Comment by mugling** [comment]
Superceded by #15072

**Pull request closed without merging**